vaccine-standby/README.md

```diff
@@ -0,0 +1,75 @@
+# COVID-19 Vaccine Standby List
+
+This Quick Deploy app is designed for public health agencies to create a COVID-19 vaccine eligibility standby list for their city or state. Residents send an SMS to a pre-configured phone number, and are asked a series of demographic questions informed by the CDC's vaccine rollout guidelines powered by a Twilio Studio Flow. Resident responses to the SMS chatbot are captured Studio Flow logs and displayed on a password protected section of the `index.html` page of this app.
+
+This data is meant to help ensure vaccinations are administered as quickly as possible as supply becomes available. With this template, your agency can create a standardized list of eligible vaccine recipients. In exchange for sharing this information, residents are notified when it is their turn to receive the vaccine.
```

Comment:
`In exchange for sharing this information, residents are notified when it is their turn to receive the vaccine.` How exactly is this intended to be accomplished? I don't see any functionality for this in the app

Reply:
This functionality is not included in this app. Good callout. I will reframe the language to make that clear.